.travis.yml

```diff
@@ -8,24 +8,19 @@ import:
 jobs:
   fast_finish: true
   include:
-    - php: 7.1
-      env:
-        - DB=MYSQL
-        - REQUIRE_INSTALLER="4.5.x-dev"
-        - PHPUNIT_TEST=1
     - php: 7.3
       env:
         - DB=MYSQL
         - PDO=1
-        - REQUIRE_INSTALLER="4.7.x-dev"
+        - REQUIRE_INSTALLER="4.10.x-dev"
         - PHPUNIT_TEST=1
         - PHPCS_TEST=1
     - php: 7.4
       env:
         - DB=MYSQL
-        - REQUIRE_INSTALLER="4.x-dev"
+        - REQUIRE_INSTALLER="4.10.x-dev"
```

Comment:
That's not going to work because there's not any `4.10` branch yet.

Reply:
Updated